var/spack/repos/builtin/packages/magics/package.py

```diff
@@ -0,0 +1,111 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Magics(Package):
+    """Magics is the latest generation of the ECMWF's Meteorological plotting
+       software MAGICS. Although completely redesigned in C++, it is intended
+       to be as backwards-compatible as possible with the Fortran interface."""
+
+    homepage = "https://software.ecmwf.int/wiki/display/MAGP/Magics"
+    url      = "https://software.ecmwf.int/wiki/download/attachments/3473464/Magics-2.29.0-Source.tar.gz"
+
+    # Maintainers of Magics do not keep tarballs of minor releases. Once the
+    # next minor released is published the previous one becomes unavailable.
+    # That is why the preferred version is the latest stable one.
+    version('2.29.4', '91c561f413316fb665b3bb563f3878d1')
+    version('2.29.0', 'db20a4d3c51a2da5657c31ae3de59709', preferred=True)
+
+    # The patch changes the hardcoded path to python in shebang to enable the
+    # usage of the first python installation that appears in $PATH
+    patch('no_hardcoded_python.patch')
```

Comment:
If the packages uses Python during installation, it should probably list python as a build dependency.